src/stories/Library/Lists/list-empty/ListEmpty.tsx

```diff
@@ -3,11 +3,12 @@ import { Links, LinksProps } from "../../links/Links";
 interface ListEmptyProps {
   text: string;
   links?: LinksProps[];
+  className: string;
```

Comment:
I think it should be optional defaulting to "". In that way you are not obliged to give it a class name. That's is a quite common pattern:

```typescript
interface ListEmptyProps {
  text: string;
  links?: LinksProps[];
  className?: string;
}

const ListEmpty = ({ text, links, className="" }: ListEmptyProps) => {